rpc_interface/rpc_interface.idl

```diff
@@ -11,7 +11,9 @@ import "wtypes.idl";
 // Interface Attributes
 //
 
+// clang-format off
 [uuid(6bef171d-7205-4b63-a1e5-d00f01e6a0c1), version(1.0), pointer_default(unique)]
+    // clang-format on
```

Comment:
is there a spacing issue we need to fix?

Reply:
i.e., the // clang-format line appears after 4 spaces. is that a editor issue?

Reply:
It's something that was done by the code formatter itself based on current configuration.